app/src/main/java/com/eveningoutpost/dexdrip/webservices/WebServiceSgv.java

```diff
@@ -184,9 +235,12 @@ public WebResponse request(String query) {
                         sensor_status_string = null;
                     }
 
+                    cachedJson.put(reading.uuid, item);
+                    withManual++;
                     reply.put(item);
                 }
 
+                Log.d(TAG, "Processed "+withCache+" cached entries and "+withManual+" manual entries");
```

Comment:
Why not use `UserError.Log.d` here as well?

Reply:
Done